swan-lake/by-example/ftp-client-file-operations/content.jsx

```
@@ -0,0 +1,334 @@
+        <Col sm={12}>
+          <pre ref={ref1}>
+            <code className="d-flex flex-column">
+              <span>{`\$ bal run ftp_client_file_operations.bal`}</span>
+            </code>
+          </pre>
+        </Col>
+      </Row>
+
+      <h2>Related links</h2>
+
+      <ul style={{ marginLeft: "0px" }} class="relatedLinks">
```

ESLint

[error] Unknown property 'class' found, use 'className' instead

(react/no-unknown-property)